lib/api/src/ticketAPI.cpp

```diff
@@ -83,336 +94,587 @@ namespace irods::administration::ticket
     }
     int ticket_manager(RxComm& conn,
                        std::string_view command,
-                       std::string_view ticket_IDentifier,
-                       std::string_view commandModifier,
+                       std::string_view ticket_Identifier,
+                       std::string_view commandModifier1,
                        std::string_view commandModifier2,
                        std::string_view commandModifier3,
                        std::string_view commandModifier4)
     {
         return ticket_manager(conn,
                               command,
-                              ticket_IDentifier,
-                              commandModifier,
+                              ticket_Identifier,
+                              commandModifier1,
                               commandModifier2,
                               commandModifier3,
                               commandModifier4,
                               false);
     }
 
-    int create_ticket(RxComm& conn, ticket_type _type, std::string_view obj_path, std::string_view ticket_name)
+    void create_ticket(RxComm& conn, ticket_type _type, std::string_view obj_path, std::string_view ticket_name)
     {
-        if (_type == ticket_type::READ) {
-            return create_read_ticket(conn, obj_path, ticket_name);
+        if (_type == ticket_type::read) {
+            ticket_manager(conn, "create", ticket_name, "read", obj_path, ticket_name, "");
         }
-        else if (_type == ticket_type::WRITE) {
-            return create_write_ticket(conn, obj_path, ticket_name);
-        }
-        return 1; // Ticket type not defined
-    }
-    int create_ticket(RxComm& conn, ticket_type _type, std::string_view obj_path)
-    {
-        if (_type == ticket_type::READ) {
-            return create_read_ticket(conn, obj_path);
+        else if (_type == ticket_type::write) {
+            ticket_manager(conn, "create", ticket_name, "write", obj_path, ticket_name, "");
         }
-        else if (_type == ticket_type::WRITE) {
-            return create_write_ticket(conn, obj_path);
+        else {
+            throw std::invalid_argument("Ticket type is not defined");
         }
```

Comment:
If the developer is using the function as intended, then we can assume this else-block is unreachable. Remember, the function is making a decision based on an enumeration.

So, let's remove it.